lambda-cloudwatch-trigger-stack.yaml

```diff
@@ -116,6 +117,9 @@ Resources:
 
                           add_permission_if_needed(event,context,lambda_arn, log_group_arn, log_group_name)
 
+                          # Unique filter name for this stack using Lambda ARN
+                          filter_name = f'LambdaSubscription_{hashlib.sha256(lambda_arn.encode()).hexdigest()[:20]}'
```

Comment:
maybe add NewRelic as prefix.

Reply:
Changed it to **NewRelicLogsLambdaSubscription_** as per naming convention for other resources.